Library/Homebrew/formula.rb

```diff
@@ -1381,12 +1381,15 @@ def stage
       mkdir_p env_home
 
       old_home, ENV["HOME"] = ENV["HOME"], env_home
+      old_java_options = ENV["_JAVA_OPTIONS"]
+      ENV.append "_JAVA_OPTIONS", "-Duser.home=#{env_home}"
```

Comment:
Mildly amusingly, `prepend` breaks where `append` works. Turns out Java will obey whichever of the same variable is passed last rather than first `¯\_(ツ)_/¯`.